src/pages/groupComparison/GroupComparisonMutationsTab.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { observer } from 'mobx-react';
+import { action, computed, observable, makeObservable } from 'mobx';
+import autobind from 'autobind-decorator';
+import GroupComparisonStore from './GroupComparisonStore';
+import _ from 'lodash';
+import {
+    MUTATIONS_NOT_ENOUGH_GROUPS_MSG,
+    MUTATIONS_TOO_MANY_GROUPS_MSG,
+} from './GroupComparisonUtils';
+import { MakeMobxView } from 'shared/components/MobxView';
+import LoadingIndicator from 'shared/components/loadingIndicator/LoadingIndicator';
+import ErrorMessage from 'shared/components/ErrorMessage';
+import { LollipopGeneSelector } from './LollipopGeneSelector';
+import GroupComparisonMutationsTabPlot from './GroupComparisonMutationsTabPlot';
+
+interface IGroupComparisonMutationsTabProps {
+    store: GroupComparisonStore;
+}
+
+@observer
+export default class GroupComparisonMutationsTab extends React.Component<
+    IGroupComparisonMutationsTabProps,
+    {}
+> {
+    constructor(props: IGroupComparisonMutationsTabProps) {
+        super(props);
+    }
+
+    readonly selectorUI = MakeMobxView({
```

Comment:
not much point to this MobxView.  might as well just put the wrapped component in tabUI